dashboard/src/components/entitypage/statusChip.js

```diff
@@ -0,0 +1,20 @@
+import { Chip } from '@mui/material';
+import React from 'react';
+
+export default function StatusChip({ status = '' }) {
+  console.log(status);
+  const chipMap = {
+    NO_STATUS: '#FFFFFF',
+    CREATED: '#FFFFFF',
```

Comment:
just curious why `NO_STATUS` and `CREATED` are both black?

Reply:
good catch (both are white) I'll quickly just set something for `created`